### ISSUE TITLE
forbid-prop-types give false positive on customObject.array attribute (not PropTypes.array)

### ISSUE THREAD

**Issue by gleb-lobastov** [yannickcr/eslint-plugin-react#1402]
Hi. Such is the bug:

```
import { formPropTypes } from 'redux-form';
class SomeComponent extends React.PureComponent {
  static propTypes = {
    // 🚨  got "Prop type `array` is forbidden" error here
    myProp: formPropTypes.array.isRequired,
  };
}

```

package version: "0.4.0"

**Comment by jseminck** [comment]
For anyone tackling this, I think this issue is similar to: https://github.com/yannickcr/eslint-plugin-react/issues/1389

We are not properly distinguishing between PropTypes coming from the `prop-types` package and custom prop-types.

**Comment by ljharb** [comment]
If this is still an issue, please post eslint config as well.
